src/lib/sepana/index.ts

```diff
@@ -0,0 +1,122 @@
+import { SepanaProjectJson, SepanaQueryResponse } from 'models/sepana'
```

Comment:
nit - we should try and avoid in index files and use them as barrel-files I think. It makes fuzzy searching a lot simpler

Reply:
honestly agree